wiki/Beatmap_ranking_procedure/en.md

```diff
@@ -55,3 +57,7 @@ Members of the BN and NAT occasionally [veto](/wiki/People/Beatmap_Nominators/Be
 Beatmaps that have passed through the qualification stage have completed the ranking procedure. They have [score leaderboards](/wiki/Ranking) and reward players with performance points.
 
 Ranked beatmaps are only unranked under exceptional circumstances when issues are found shortly after they reach Ranked status.
+
+## Notes
+
+[^hybrid-sets]: For beatmaps with difficulties across multiple game modes, the main game mode is the mode with the most difficulties in total or — if the amount is the same — by the host. Non-primary game modes require only one nomination by a full BN. Hybrid Beatmap Nominators who nominate these sets will increase the progress for all game modes they belong to.
```

Comment:
this doesn't mention the 3rd fallback where the primary mode can be any mode on the set if the first 2 conditions aren't satisfied.

maybe instead of a footnote you could better explain this in a paragraph in `## Qualification`, that way you could do a proper list for all the conditions. thoughts?

Reply:
Was in the original PR. Can be done, have the original wording in https://github.com/ppy/osu-wiki/pull/13211/commits/dc26c950767935fba84fc83cc639ba501668dedc and prior commits before https://github.com/ppy/osu-wiki/pull/13211/commits/32b49de3ca267d04397d5a584671ff9dcff109fe.

Feel free to review those, can just rebase.

Reply:
As a suggestion:

`Should both conditions result in the same amount, then any game mode can be the main game mode.`

Reply:
Will apply prior blocks and dismiss [this](https://github.com/ppy/osu-wiki/pull/13211#discussion_r2072253203) then.

Reply:
@Hiviexd See latest commit.